lib/capistrano/tasks/wp.cap

```diff
@@ -12,6 +24,46 @@ namespace :wp do
 
   namespace :setup do
 
+    # Generates a random password and stores it as :wp_pass
+    task :generate_password do
+      o = [('a'..'z'), ('A'..'Z')].map { |i| i.to_a }.flatten
+      password = (0...18).map { o[rand(o.length)] }.join
+      set :wp_pass, password
+    end
+
+    # Installs WordPress
+    task :install
+      config = Config.new
```

Comment:
At one point, I thought it would help to be able to pass additional options into the `Config` initializer, but it turned out not to help. So I deleted that parameter. The other way to solve it would have been to provide a default value (an empty Hash):

```ruby
def initialize(options = {})
```